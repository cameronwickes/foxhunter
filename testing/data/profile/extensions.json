{"schemaVersion":35,"addons":[{"id":"{randomID1}","syncGUID":"{randomGUID1}","version":"1.10.0","type":"extension","loader":null,"installOrigins":null,"manifestVersion":2,"optionsURL":null,"optionsType":null,"optionsBrowserStyle":true,"aboutURL":null,"defaultLocale":{"name":"EX","description":"A fake extension from Mozilla","creator":null,"developers":null,"translators":null,"contributors":null},"visible":true,"active":true,"userDisabled":false,"appDisabled":false,"embedderDisabled":false,"installDate":1627290541559,"updateDate":1627290541559,"applyBackgroundUpdates":1,"path":"/random/path/to/xpi.xpi","skinnable":false,"sourceURI":"https://addons.mozilla.org/firefox/downloads/file/random/random.xpi","releaseNotesURI":null,"softDisabled":false,"foreignInstall":false,"strictCompatibility":true,"locales":[],"targetApplications":[{"id":"toolkit@mozilla.org","minVersion":"48.0","maxVersion":"*"}],"targetPlatforms":[],"signedState":2,"signedDate":1621953325000,"seen":true,"dependencies":[],"incognito":"spanning","userPermissions":{"permissions":["webRequest", "webRequestBlocking"],"origins":[]},"optionalPermissions":{"permissions":[],"origins":[]},"icons":null,"iconURL":null,"blocklistState":0,"blocklistURL":null,"startupData":null,"hidden":false,"installTelemetryInfo":{"source":"amo","sourceURL":"https://addons.mozilla.org/en-GB/firefox/addon/medium-unlimited-read-for-free/","method":"amWebAPI"},"recommendationState":null,"rootURI":"jar:file:///random/path/to/xpi.xpi","location":"app-profile"},{"id":"{randomID2}","syncGUID":"{randomGUID1}","version":"1.10.0","type":"extension","loader":null,"installOrigins":null,"manifestVersion":2,"optionsURL":null,"optionsType":null,"optionsBrowserStyle":true,"aboutURL":null,"defaultLocale":{"name":"NMI","description":"A fake extension not from Mozilla","creator":null,"developers":null,"translators":null,"contributors":null},"visible":true,"active":true,"userDisabled":false,"appDisabled":false,"embedderDisabled":false,"installDate":1627290541559,"updateDate":1627290541559,"applyBackgroundUpdates":1,"path":"/random/path/to/xpi.xpi","skinnable":false,"sourceURI":"https://random.com/random.xpi","releaseNotesURI":null,"softDisabled":false,"foreignInstall":false,"strictCompatibility":true,"locales":[],"targetApplications":[{"id":"toolkit@mozilla.org","minVersion":"48.0","maxVersion":"*"}],"targetPlatforms":[],"signedState":2,"signedDate":1621953325000,"seen":true,"dependencies":[],"incognito":"spanning","userPermissions":{"permissions":["webRequest", "webRequestBlocking"],"origins":[]},"optionalPermissions":{"permissions":[],"origins":[]},"icons":null,"iconURL":null,"blocklistState":0,"blocklistURL":null,"startupData":null,"hidden":false,"installTelemetryInfo":{"source":"amo","sourceURL":"https://addons.mozilla.org/en-GB/firefox/addon/medium-unlimited-read-for-free/","method":"amWebAPI"},"recommendationState":null,"rootURI":"jar:file:///random/path/to/xpi.xpi","location":"app-profile"},{"id":"{randomID3}","syncGUID":"{randomGUID1}","version":"1.10.0","type":"extension","loader":null,"installOrigins":null,"manifestVersion":2,"optionsURL":null,"optionsType":null,"optionsBrowserStyle":true,"aboutURL":null,"defaultLocale":{"name":"PRM","description":"A fake extension with interesting permissions from Mozilla","creator":null,"developers":null,"translators":null,"contributors":null},"visible":true,"active":true,"userDisabled":false,"appDisabled":false,"embedderDisabled":false,"installDate":1627290541559,"updateDate":1627290541559,"applyBackgroundUpdates":1,"path":"/random/path/to/xpi.xpi","skinnable":false,"sourceURI":"https://addons.mozilla.org/firefox/downloads/file/random/random.xpi","releaseNotesURI":null,"softDisabled":false,"foreignInstall":false,"strictCompatibility":true,"locales":[],"targetApplications":[{"id":"toolkit@mozilla.org","minVersion":"48.0","maxVersion":"*"}],"targetPlatforms":[],"signedState":2,"signedDate":1621953325000,"seen":true,"dependencies":[],"incognito":"spanning","userPermissions":{"permissions":["webRequest", "downloads", "downloads.open", "background", "cookies"],"origins":[]},"optionalPermissions":{"permissions":[],"origins":[]},"icons":null,"iconURL":null,"blocklistState":0,"blocklistURL":null,"startupData":null,"hidden":false,"installTelemetryInfo":{"source":"amo","sourceURL":"https://addons.mozilla.org/en-GB/firefox/addon/medium-unlimited-read-for-free/","method":"amWebAPI"},"recommendationState":null,"rootURI":"jar:file:///random/path/to/xpi.xpi","location":"app-profile"}]}