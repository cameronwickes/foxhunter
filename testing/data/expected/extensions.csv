name,URL,permissions
EX,https://addons.mozilla.org/firefox/downloads/file/random/random.xpi,"['webRequest', 'webRequestBlocking']"
NMI,https://random.com/random.xpi,"['webRequest', 'webRequestBlocking']"
PRM,https://addons.mozilla.org/firefox/downloads/file/random/random.xpi,"['webRequest', 'downloads', 'downloads.open', 'background', 'cookies']"
